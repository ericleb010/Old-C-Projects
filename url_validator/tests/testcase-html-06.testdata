-d 0 http://itishardtofindaninvaliddomain.com
