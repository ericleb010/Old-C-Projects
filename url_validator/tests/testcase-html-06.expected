
Checking http://itishardtofindaninvaliddomain.com ... **BROKEN**
